core/tests/puppeteer-acceptance-tests/blog-post-unpublish.js

```diff
@@ -0,0 +1,18 @@
+const e2eBlogPostAdmin = require("./utility-functions/blogAdminUtils.js");
+const testConstants = require("./utility-functions/testConstants.js");
+
+
+const blogDashboard = testConstants.URLs.BlogDashboard;
+
+async function unpublishBlogAsBlogAdmin() {
+  const blogPostAdmin = await new e2eBlogPostAdmin();
+  await blogPostAdmin.getInitialized();
+  
+  await blogPostAdmin.signInWithEmail("testadmin@example.com");
+  await blogPostAdmin.goto(blogDashboard);
+  await blogPostAdmin.unpublishBlog();
```

Comment:
unpublishBlogPost

Wouldn't this need an argument to say "which one"? E.g. unpublishBlogPostByTitle('Annual Report')

Ditto for delete and create, these may need a little bit of parameterization so that they work generically.

Reply:
done